src/ui/atoms/toast/Toast.tsx

```diff
@@ -62,9 +63,13 @@ export const Toast: React.FC<ToastProps> = ({
         )}
         {description && (
           <ToastPrimitive.Description asChild className="okp4-toast-description">
-            <Typography as="span" color="highlighted-text" fontSize="small" fontWeight="light">
-              {description}
-            </Typography>
+            {typeof description === 'string' ? (
```

Comment:
I'd like to see this condition wrapped into a `typeof` guard function.

Reply:
I've put the guard function in utils as it is a general purpose function